opendocdb-cts/main.go

```diff
@@ -140,7 +140,7 @@ func runCommand(ctx context.Context, l *slog.Logger) error {
 			l.InfoContext(ctx, name+": PASSED")
 			results = append(results, testresult.TestSuiteResult{Name: name, Passed: true})
 		} else {
-			l.ErrorContext(ctx, name+": FAILED\n"+err.Error())
+			l.ErrorContext(ctx, name+": FAILED\n::group::Error\n"+err.Error()+"\n::endgroup::")
```

Comment:
Well, that makes the local output worse.

The issue specifically points out https://pkg.go.dev/github.com/sethvargo/go-githubactions#Group. I think this library does nothing if it is not running on GitHub Actions.

Reply:
I'm not sure if the usage of the library gives any profits at this point.
Those helper functions just print `group` and `endgroup` into the underlying writer, which by default is just set to `os.Stdout`.
It doesn't perform any checks if we're running on CI or not.

I've added check for `GITHUB_ACTIONS` env variable. Please let me know if we want to use the library in those circumstances.